docs/tutorials/creating-a-simulation.rst

```diff
@@ -14,53 +14,36 @@ and wire them together.
 Adding Devices
 --------------
 
-In order to be included in a simulation, tickit devices must be nested within a
-`DeviceSimulation` component. At the top level, tickit simulations comprise a list of
-components - denoted in YAML by ``-``. As such we shall begin by adding an empty
-`DeviceSimulation` as the first element of our list, as:
+In order to be included in a simulation, tickit devices must have a `ComponentConfig`
+dataclass associated with them. This defines the device to be used, as well as any
+adapters to allow the device to be externally controlled. At the top level, tickit 
+simulations comprise a list of these components - denoted in YAML by ``-``.
```

Comment:
I'm not sure what you mean here about denoting by `-`?

Reply:
Oh that was in the old docs, i just left it in because i assumed it made sense to whoever wrote that originally :laughing:

Reply:
Paging @garryod